llvm/test/Transforms/InstCombine/getelementptr.ll

```diff
@@ -680,7 +680,7 @@ entry:
 define i32 @test28() nounwind  {
 ; CHECK-LABEL: @test28(
 ; CHECK-NEXT:  entry:
-; CHECK-NEXT:    [[ORIENTATIONS:%.*]] = alloca [1 x [1 x %struct.x]], align 8
+; CHECK-NEXT:    [[ORIENTATIONS:%.*]] = alloca [1 x [1 x [[STRUCT_X:%.*]]]], align 8
```

Comment:
I don't know why this line changed. Is this a harmless change? I'm not 100% sure, but it looks like a cosmetic (variable renaming) change to me.